host/contracts/contracts.go

```diff
@@ -164,6 +164,17 @@ var (
 	ErrContractExists = errors.New("contract already exists")
 )
 
+// NewUsageFromRPCCost returns a new Usage from the given RPCCost.
+func NewUsageFromRPCCost(c rhp2.RPCCost) Usage {
```

Comment:
This feels like it should be unexported in `rhp/v2` since it won't be used anywhere else.